src/chocolatey.resources/helpers/functions/UnInstall-ChocolateyZipPackage.ps1

```diff
@@ -50,7 +50,7 @@ param(
     $zipContentFile
     $zipContents=get-content $zipContentFile
     foreach ($fileInZip in $zipContents) {
-      remove-item "$fileInZip" -ErrorAction SilentlyContinue
+      remove-item -Path "$fileInZip" -ErrorAction SilentlyContinue -Recurse -Force

```

Comment:
I am not sure this will fix the issue - You should instead employ a check to see if it is a directory or file like https://github.com/chocolatey/choco/blob/master/nuget/chocolatey/tools/chocolateysetup.psm1#L235-L244